README.md

```diff
@@ -1,2 +1,70 @@
-# portfolioskeleton1
-Mobile Portfolio Skeleton task 1
+
+# Project Name: Portfolio Skeleton Project
+
+> header and and headline section 
+Created new branch according to GitHub flow rules
+Added a .gitignore file
+setup a linters for html and css and made sure that files not relevant are ignored by git.
+Added a descriptive README file 
+Added a header and headline section in the index.html file  of the project 
+Added css styling to the html sections 
+
```

Comment:
- [ ] There is one important section missing here and that is the `Screenshot` section. This is where you add a screenshot of the landing page of your website or web app **so as to give visitors a quick preview of your website**. I think it would be better if you add a screenshot section here 😃

Reply:
Noted with thanks